examples/embeddable_examples/public/react_embeddables/data_table/create_data_table_action.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { apiIsPresentationContainer } from '@kbn/presentation-containers';
+import { EmbeddableApiContext } from '@kbn/presentation-publishing';
+import { IncompatibleActionError, UiActionsStart } from '@kbn/ui-actions-plugin/public';
+import { ADD_DATA_TABLE_ACTION_ID, DATA_TABLE_ID } from './constants';
+
+// -----------------------------------------------------------------------------
+// Create and register an action which allows this embeddable to be created from
+// the dashboard toolbar context menu.
+// -----------------------------------------------------------------------------
+export const registerCreateDataTableAction = (uiActions: UiActionsStart) => {
+  uiActions.registerAction<EmbeddableApiContext>({
+    id: ADD_DATA_TABLE_ACTION_ID,
+    getIconType: () => 'tableDensityNormal',
+    isCompatible: async ({ embeddable }) => {
+      return apiIsPresentationContainer(embeddable);
+    },
+    execute: async ({ embeddable }) => {
+      if (!apiIsPresentationContainer(embeddable)) throw new IncompatibleActionError();
+      embeddable.addNewPanel(
+        {
+          panelType: DATA_TABLE_ID,
+        },
+        true
+      );
+    },
+    getDisplayName: () =>
```

Comment:
lets remove i18n translations from examples.
1. Examples target developers and all development is done in English
2. Creating translations cost money
3. Translations bloat bundle size

Reply:
I like keeping the `i18n` in examples because I'd like it to show our best practices and I'd like the code to look / work as close as possible to the real world code.

I agree that all development on Kibana is done in English, but I'm pretty sure usages in `examples` don't get translated - as they shouldn't:
There are 0 instances of `embeddableExamples` in the translation files, and some `i18n` usages have been around for over 4 years. `embeddableExamples.searchableListContainer.displayName` and `embeddableExamples.searchableListContainer.displayName` are roughly that old.

As for the bundle sizes, this only applies when user is running Kibana with `--run-examples` - which only developers should be doing.